Runtime/Native/iOS/NativeClient.cs

```diff
@@ -63,7 +63,7 @@ public NativeClient(BacktraceConfiguration configuration, BacktraceBreadcrumbs b
             }
             if (_configuration.CaptureNativeCrashes)
             {
-                HandleNativeCrashes(clientAttributes, attachments);
+                HandleNativeCrashes(clientAttributes, attachments, configuration.ClientSideUnwinding);
```

Comment:
I think we don't need to pass it there via parameter - we have an access to the _configuration object anyway in the method body.